controllers/humiocluster_controller.go

```diff
@@ -399,6 +406,60 @@ func (r *HumioClusterReconciler) ensureExtraKafkaConfigsConfigMap(ctx context.Co
 	return nil
 }
 
+// validateEnvVarSource validates that a envVarSource exists if the environmentVariablesSource is specified
```

Comment:
This comment looks a little weird to me. First word also doesn't match the function name.

Reply:
Ah, this was due to a refactor and I didn't update the comment. Fixed.